src/main/java/net/javadiscord/javabot/util/WebhookUtil.java

```diff
@@ -63,9 +63,7 @@ public static void ensureWebhookExists(@NotNull IWebhookContainer channel, @NotN
 			}
 		};
 		channel.retrieveWebhooks().queue(webhooks -> {
-			Optional<Webhook> hook = webhooks.stream()
-					.filter(webhook -> webhook.getChannel().getIdLong() == channel.getIdLong())
-					.filter(wh -> wh.getToken() != null).findAny();
+			Optional<Webhook> hook = webhooks.stream().filter(webhook -> webhook.getChannel().getIdLong() == channel.getIdLong()).filter(wh -> wh.getToken() != null).findAny();
```

Comment:
Don't convert streams/chained calls to a single line when they are split over multiple lines for readability.